cellxgene_schema_cli/scripts/gene_processing.py

```diff
@@ -248,13 +253,29 @@ def process_gene_infos(self, gene_infos: dict) -> None:
                     )
                 self.write_gzip(output_to_print, new_file)
 
-                if self.digest(new_file) == self.digest(previous_ref_filepath):
-                    print("New gene reference is identical to previous gene reference", gene_info_description)
-                    os.remove(new_file)
-                else:
-                    print("generating gene reference diff for", gene_info_description)
-                    self.generate_gene_ref_diff(gene_info_description, new_file, previous_ref_filepath)
+                # Rename new_genes_{filename} to genes_{filename} if this is the first time we're introducing a new gene file type
+                if not os.path.exists(previous_ref_filepath):
                     os.replace(new_file, previous_ref_filepath)
+
+                    new_ref_gene_ids = set()
```

Comment:
the gencode *_diff files are a list of which genes have been deprecated in between versions (see `generate_gene_ref_diff`), for use by the migrate script. So in the case of first-time gencode files, we should just ignore generating it, since there is nothing to deprecate